packages/client/screens/TagsSettingsScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { View } from "../components/Themed";
+import { Badge, Box, Center, HStack, Text } from "native-base";
+import { StyleSheet } from "react-native";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import { falTags } from "../components/icons/light/falCircleArrowDownLeft copy";
+import { AuthContext } from "../components/contexts/AuthContext";
+import { TransactionTag } from "@cryptify/common/src/domain/entities/TransactionTag";
+import { TagsGateway } from "../gateways/tags_gateway";
+
+export default function TagsSettingsScreen() {
+    const tagsGateway = new TagsGateway();
+
+    const { token, user } = React.useContext(AuthContext);
+
+    const [tags, setTags] = React.useState<TransactionTag[]>([]);
+
+    React.useEffect(() => {
+        (async () => {
+            const tags = await tagsGateway.findAlTags({ id: user.id }, token);
+            setTags(tags);
+        })();
+    });
+
+    return tags.length == 0 ? (
+        <View style={styles.view}>
+            <Center alignItems="center" marginY="auto">
+                <Box marginTop="-10px"></Box>
+                <FontAwesomeIcon icon={falTags} style={styles.tagIcon} size={56} />
+                <Text style={styles.noTagsText}>You do not have any tags.</Text>
+            </Center>
+        </View>
+    ) : (
+        <View style={styles.view}>
+            <Box marginTop="10px"></Box>
+            <HStack flexWrap="wrap">
+                {tags.map((tag, i) => (
+                    <Badge rounded="md" color="gray.100" style={styles.badge} key={i}>
```

Comment:
It does not seem like the text semibold font weight is working. Try adding the styling directly to the badge component itself (badge text styling is handled by the badge theme: https://github.com/GeekyAnts/NativeBase/blob/master/src/theme/components/badge.ts)

```suggestion
                    <Badge rounded="md" color="gray.100" style={styles.badge} key={i} _text={{ fontSize: "subheadline", fontWeight:"semibold"}}>
```

If this does not work, try the following comment.